src/form/input/number/NumberInput.tsx

```diff
@@ -50,7 +50,14 @@ const NumberInput = React.forwardRef<HTMLInputElement, NumberInputProps>((props,
   });
 
   return (
-    <Input ref={ref} type={"text"} onChange={handleChange} value={finalValue} {...rest} />
+    <Input
+      ref={ref}
+      customClassName={"number-input"}
+      type={"text"}
+      onChange={handleChange}
+      value={finalValue}
+      {...rest}
+    />
```

Comment:
I'm afraid it won't be enough.
- If I provide `customClassName` to `NumberInput`, this class name will be overridden because `...rest` has `customClassName`. Although we don't have any specific style for the number input, we should keep that class name.
- I think the `NumberInput` should be able to take custom class names, so we should have both.

Here is my suggestion:

```suggestion
    <Input
      ref={ref}
      customClassName={classNames("number-input", customClassName)}
      type={"text"}
      onChange={handleChange}
      value={finalValue}
      {...rest}
    />
```